frontend/src/components/FileList.vue

```diff
@@ -98,6 +109,12 @@ export default {
     vertical-align: middle;
   }
 
+  #uploadRow {
+    background-color: #f8f9fa;
+    width: 100vw;
+    height: 100vh;
```

Comment:
`100vh` creates an unecessary scroll bar. (because the height becomes the view height not really taking into consideration that the border box of this row starts like 150px lower from the top)

I tried to fix this myself could you check it out at this branch/commit and tell me what you think?
https://github.com/amosproj/amos2021ws03-teams-to-nextcloud/commit/3bed690f29d0f041d3e9b8fb5264e433413540e0